src/components/Comment/Comment.tsx

```diff
@@ -488,17 +496,22 @@ export const Comment = ({
                     <>
                       {/* If user is not logged in we link to the login page */}
                       {user ? (
-                        <button
-                          onClick={() => setCommentBeingRepliedTo(comment)}
+                        <div
                           className={cx(
-                            flexRowStyles,
-                            removePaddingLeft,
-                            commentControlsButtonStyles
+                            buttonHeightOverrides,
+                            commentControlsButtonWrapperStyles,
+                            removeButtonPaddingLeft
                           )}
                         >
-                          <ReplyArrow />
-                          Reply
-                        </button>
+                          <Button
+                            priority="tertiary"
+                            size="small"
+                            onClick={() => setCommentBeingRepliedTo(comment)}
+                          >
+                            <ReplyArrow />
+                            Reply
+                          </Button>
+                        </div>
```

Comment:
Button has the `icon` and `iconPosition` props we can use here

Reply:
I tired this, but it looks like a bunch of unwanted styling (colour and padding) applied to the SVG, we can override them as-well knowing we are doing something similar to the Button styles

<img width="86" alt="Screenshot 2020-04-06 at 13 42 44" src="https://user-images.githubusercontent.com/8831403/78559660-b03dc400-780c-11ea-88b7-afa17d1f816e.png">

Reply:
It's because it's expecting an icon of a certain size. If you used one of the Source icons it works fine. Check out `svgOverrides`